src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -154,9 +155,8 @@ private void Apply_Click(object sender, RoutedEventArgs e)
                 }
                 else
                 {
-                    model.Apply();
+                    model.Persist();
                 }
-

```

Comment:
SA1513
Closing brace should be followed by blank line

Reply:
Done